packages/docs/src/components/layout.js

```diff
@@ -38,70 +32,82 @@ export default props => {
     setMode(next)
   }
 
+  const title = [
+    props._frontmatter ? props._frontmatter.title : false,
+    'Theme UI'
+  ]
+    .filter(Boolean)
+    .join(' — ')
+
   return (
     <Styled.root>
+      <Helmet>
```

Comment:
Just conceptually thinking here, but I find it nice to keep the head component in `wrapPageElement`

Reply:
I don't think we can do that because `wrapPageElement` doesn't receive the props we need (`_frontmatter`). I've created a `Head` component to keep the `layout` component cleaner. What do you think?